packages/notifi-frontend-client/lib/NotifiFrontendClient.ts

```diff
@@ -73,4 +88,49 @@ export class NotifiFrontendClient {
 
     await Promise.all([saveAuthorizationPromise, saveRolesPromise]);
   }
+
+  async beginLoginViaTransaction({
+    walletBlockchain,
+    walletAddress,
+  }: BeginLoginProps): Promise<BeginLogInByTransactionResult> {
+    const { dappAddress } = this._configuration;
+
+    const result = await this._service.beginLogInByTransaction({
+      walletAddress: walletAddress,
+      walletBlockchain: walletBlockchain,
+      dappAddress,
+    });
+
+    const nonce = result.beginLogInByTransaction.nonce;
```

Comment:
these objects are nullable, but i don't see null-safe access

Can you share your typescript configuration with me? I think something's not right in the IDE

Reply:
During our call, we confirmed the TS config is correct and the IDE is not advising on implementing chaining since the object cannot be nullable.